sqlx-core/src/logger.rs

```diff
@@ -22,6 +22,10 @@ impl<'q> QueryLogger<'q> {
         self.rows += 1;
     }
 
+    pub(crate) fn increment_rows_by(&mut self, n: usize) {
```

Comment:
This should probably be `u64` since there's not really any sane relation to pointer arithmetic here.